Docs/source/usage/faq.rst

```diff
@@ -6,17 +6,28 @@ FAQ
 This section lists frequently asked usage questions.
 
 
-What is MPI thread support level?
----------------------------------
+What is "MPI initialized with thread support level ..."?
+--------------------------------------------------------
 
-We report this in output on startup together with other information.
+When we start up WarpX, we report a couple of information on used MPI processes across parallel compute processes, CPU threads or GPUs and further capabilities.
+For instance, a parallel, multi-process, multi-threaded CPU run could output::
 
-That is the `MPI support for threaded execution <https://www.mpich.org/static/docs/v3.1/www3/MPI_Init_thread.html>`__, e.g., with OpenMP or system threads.
+   MPI initialized with 4 MPI processes
+   MPI initialized with thread support level 3
+   OMP initialized with 8 OMP threads
+   AMReX (22.10-20-g3082028e4287) initialized
+   ...
 
+The 1st line is the number of parallel MPI processes (also called *MPI ranks*).
+
+The 2nd line reports on the `support level of MPI functions to be called from threads <https://www.mpich.org/static/docs/v3.1/www3/MPI_Init_thread.html>`__.
 We currently only use this for optional, :ref:`async IO with AMReX plotfiles <running-cpp-parameters-diagnostics>`.
 In the past, requesting MPI threading support had performance penalties, but we have not seen such anymore on recent systems.
 Thus, we request it by default but you can overwrite it with a compile time option if it ever becomes needed.
 
```

Comment:
"Edit" is greyed out for me, so I'll do it here. 😄 

> The 2nd line reports on the `support level of the MPI library <https://www.mpich.org/static/docs/v3.1/www3/MPI_Init_thread.html>`__.
> A support level of 4 means WarpX has been compiled with MPI_THREAD_MULTIPLE support.
>  We currently only use this for optional, :ref:`async IO with AMReX plotfiles <running-cpp-parameters-diagnostics>`.
>  In the past, requesting MPI threading support had performance penalties, but we have not seen such anymore on recent systems.
>  Thus, we request MPI_THREAD_MULTIPLE by default but you can overwrite it with a compile time option if it ever becomes needed.

Reply:
Good point,you mean `amrex.async_out_nfiles`? Already linked, but now also mentioned.

Reply:
n_files just says the number of files to be used, right? It doesn't turn off Async I/O and THREAD_MULTIPLE initialization. 

(Hence why just pointing to the appropriate list of all the options might be better... then we also aren't updating in two places at once. 😄)

Reply:
Which options am I missing? Build or run-time?

Reply:
To turn THREAD_MULTIPLE off (which I think is the discussion being had here),

https://warpx.readthedocs.io/en/latest/install/cmake.html#build-options

WARPX_MPI_THREAD_MULTIPLE=OFF.

I don't know how async io works/responds if you have it on but THREAD_MULTIPLE off. Been a while. Might need to include that being off too (which I think is runtime).

Reply:
Ah ok, this flag is mentioned and further linked i the runtikme inputs that are linked. Let me finalize.